.github/workflows/trackLLVMChanges.yml

```diff
@@ -0,0 +1,81 @@
+name: Track LLVM Changes
+
+on:
+  schedule:
+    # 10:00 AM UTC is 3AM Pacific Time
+    - cron: '0 10 * * *'
+  # Run this workflow on pull requests which change this workflow
+  pull_request: 
+    paths:
+      - .github/workflows/trackLLVMChanges.yml
+  workflow_dispatch:
+
+jobs:
+  track-llvm-changes:
+    name: Track LLVM Changes
+    runs-on: ubuntu-20.04
+    steps:
+      # Clone the CIRCT repo and its submodules. Do shallow clone to save clone
+      # time.
+      - name: Get CIRCT
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+          submodules: "true"
+
+      - name: Configure Project
+        run: |
+          mkdir build
+          cd build
+          cmake ../llvm/llvm \
+            -DLLVM_USE_LINKER=lld \
+            -DCMAKE_C_COMPILER=clang \
+            -DCMAKE_CXX_COMPILER=clang++ \
+            -DCMAKE_RULE_MESSAGES=OFF \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DBUILD_SHARED_LIBS=ON \
+            -DLLVM_ENABLE_PROJECTS=mlir \
+            -DLLVM_TARGETS_TO_BUILD=host \
+            -DLLVM_BUILD_EXAMPLES=OFF \
+            -DLLVM_INSTALL_UTILS=OFF \
+            -DLLVM_ENABLE_OCAMLDOC=OFF \
+            -DLLVM_ENABLE_BINDINGS=OFF \
+            -DLLVM_ENABLE_ASSERTIONS=ON \
+            -DLLVM_EXTERNAL_PROJECTS=circt \
+            -DLLVM_EXTERNAL_CIRCT_SOURCE_DIR=$PWD/..
```

Comment:
Is this close enough to https://github.com/llvm/circt/blob/main/utils/build-llvm.sh that `build-llvm.sh` can be re-used here?

Reply:
#No unfortunately build-llvm installs llvm and does not build circt. We could pass a script to git bisect run, but that feels like it would be more complex and prone to failure